master/reference/calicoctl/resources/workloadendpoint.md

```diff
@@ -60,24 +61,24 @@ spec:
 
 #### Metadata
 
-| Field       | Description                 | Accepted Values   | Schema | Default    |
-|-------------|-----------------------------|-------------------|--------|------------|
-| name      | The name of this workload endpoint resource. Reqired. |  Alphanumeric string with optional `.`, `_`, `-`, or `/` | string | |
-| namespace | Namespace provides an additional qualification to a resource name. | | map | "default" |
+| Field     | Description                                                        | Accepted Values                                          | Schema | Default   |
+|-----------|--------------------------------------------------------------------|----------------------------------------------------------|--------|-----------|
+| name      | The name of this workload endpoint resource. Required.             |  Alphanumeric string with optional `.`, `_`, `-`, or `/` | string |           |
+| namespace | Namespace provides an additional qualification to a resource name. |                                                          | map    | "default" |
 
 #### Spec
 
-| Field       | Description                 | Accepted Values   | Schema | Default    |
-|-------------|-----------------------------|-------------------|--------|------------|
-| workload       | The name of the workload to which this endpoint belongs. |  | string |
-| orchestrator   | The orchestrator that created this endpoint.  |  | string |
-| node           | The node where this endpoint resides.   |  | string |
-| labels         | A set of labels to apply to this endpoint.              |  | map |
-| ipNetworks    | The CIDRs assigned to the interface. |  | List of strings |
-| profiles      | List of profiles assigned to this endpoint.             | | List of strings |
-| interfaceName | The name of the host-side interface attached to the workload. | | string |
-| mac           | The source MAC address of traffic generated by the workload. | | IEEE 802 MAC-48, EUI-48, or EUI-64 |
-| ports         | List on named ports that this workload exposes. | | List of [EndpointPorts](#endpointport) |
+| Field          | Description                                                   | Accepted Values | Schema                                 | Default |
+|----------------|---------------------------------------------------------------|-----------------|----------------------------------------|---------|
```

Comment:
Fine to do, but it's really just a pain to try and keep all the spacing lined up.

Reply:
I know but it's otherwise very hard to read if the spacing is incorrect.